processor/servicegraphprocessor/processor.go

```diff
@@ -204,7 +206,7 @@ func (p *processor) aggregateMetrics(ctx context.Context, td ptrace.Traces) (err
 						e.ClientService = serviceName
 						e.ClientLatencySec = float64(span.EndTimestamp()-span.StartTimestamp()) / float64(time.Millisecond.Nanoseconds())
 						e.Failed = e.Failed || span.Status().Code() == ptrace.StatusCodeError
-						p.upsertDimensions(e.Dimensions, rAttributes, span.Attributes())
+						p.upsertDimensions(clientKind, e.Dimensions, rAttributes, span.Attributes())
```

Comment:
Is this going to require a change to current consumers of this component? In other words, is this a breaking change?

Reply:
no, it's not a breaking change.

Reply:
It is a break change indeed, I think, the users need to change the `metrics name` to query data.